app/data/config/watcher-mobymask-v2/mobymask-params.env

```diff
@@ -3,6 +3,12 @@ WATCHER_HOST="mobymask-watcher-server"
 WATCHER_PORT=3001
 APP_WATCHER_URL="http://localhost:3001"
 
+# Base URI for mobymask-app (used for generating invite)
+MOBYMASK_APP_BASE_URI="http://127.0.0.1:3002/#"
```

Comment:
nice ... this looks like the fix I was searching for, wanting to get a demo working on a cloud machine